app/src/main/res/layout/help.xml

```diff
@@ -9,7 +9,8 @@
   <ScrollView
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/blue_grey">
+    android:background="@color/blue_grey"
+      android:fillViewport="true">
```

Comment:
Check the indentation here.

Reply:
That's for a different purpose, I mistakenly commited that on this branch, will correct it.
The problem is with the theme of the alert dialog. Someone help me with the right theme for dark mode for alert dialog.